hash_to_animals.py

```diff
@@ -12,18 +11,20 @@ def get_animal_names(filename="animals.json"):
 def number_to_animal(animals):
     return {i: animals[i] for i in range(128)}
 
-
-if __name__ == '__main__':
-    if len(sys.argv) < 2:
-        print("not enough arguments")
-        sys.exit(0)
-
-    message = sys.argv[1]
+def hash_input(msg):
     m = hashlib.sha256()
-    m.update(message.encode('utf-8'))
-
+    m.update(msg.encode('utf-8'))
     animals = get_animal_names()
     n_to_a = number_to_animal(animals)
     animal_hash = [n_to_a[byte & 0b01111110] for byte in m.digest()[:3]]
 
     print("-".join(animal_hash))
+
+
+if __name__ == '__main__':
+    if len(sys.argv) < 2:
```

Comment:
what if the number of arguments are > 3? Maybe consider using a better python CLI library like [argparse](https://docs.python.org/3/library/argparse.html) or [python-fire](https://github.com/google/python-fire)?